routers/cities.py

```diff
@@ -0,0 +1,46 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.orm import Session
+from dependencies import get_db
+import crud
+import schemas
+
+router = APIRouter()
+
+
+@router.post("/cities/", response_model=schemas.City, status_code=201)
+def create_city(city: schemas.CityCreate, db: Session = Depends(get_db)):
+    return crud.city.create_city(db=db, city=city)
+
+
+@router.get("/cities/", response_model=list[schemas.City])
+def read_cities(db: Session = Depends(get_db)):
+    return crud.city.get_cities(db=db)
+
+
+@router.get("/cities/{city_id}", response_model=schemas.City)
```

Comment:
```suggestion
@router.get("/cities/{city_id}/", response_model=schemas.City)
```
please fix everywhere